rules/opentitan/exec_env.bzl

```diff
@@ -163,6 +164,17 @@ def exec_env_common_attrs(**kwargs):
         #    executable = True,
         #    cfg = "exec",
         #),
+        #"rom_scramble_tool": attr.label(
```

Comment:
Why is this commented? Is it due to the maybe-bazel-bug you ran into?

Reply:
That is why it was commented, but I was able to ask a bazel expert about it and he showed me how to correct it (see https://github.com/cfrantz/bugs/commit/77b65b58180868227eaeabbf59337eb5cef41e69 for the details).